src/api/auth.ts

```diff
@@ -118,6 +118,12 @@ export function getTeamUsers(team: string, organization: string, auth: any): any
   return users;
 }
 
+export function checkPackageName(pkgName: string): boolean {
+  let regexStart = /^[a-z0-9~()'!*:@,;+?-]{1}/g;
```

Comment:
I know, npm documentation https://docs.npmjs.com/files/package.json says `the name can't contain any non-URL-safe characters`, which means that `$-_.+!*'(),` should be okay... Although you're right, it really shouldn't contain `~:?` I don't know of course we can be more strict than npm about package names, what do you think?